packages/common/src/FindContractVersion.js

```diff
@@ -12,7 +12,7 @@ async function findContractVersion(contractAddress, web3) {
   assert(contractAddress, "Contract address must be provided");
   const contractCode = await web3.eth.getCode(contractAddress);
   const contractCodeHash = web3.utils.soliditySha3(contractCode);
-  return versionMap[contractCodeHash] || {};
+  return versionMap[contractCodeHash] || { contractCodeHash, contractAddress };
```

Comment:
helps a bit with debugging. this is changed somewhat in https://github.com/UMAprotocol/protocol/pull/2477 as well but I will leave that primary change for that PR.